coveralls/api.py

```diff
@@ -45,20 +45,26 @@ def __init__(self, token_required=True, **kwargs):
         repo_token = self.config.get('repo_token') or file_config.get('repo_token', None)
         if repo_token:
             self.config['repo_token'] = repo_token
+        service_name = self.config.get('service_name') or file_config.get('service_name', None)
```

Comment:
I don't see where we set `self.config['service_name']` to `self.default_client` by default. Should we do it here?